templates/configfile.j2

```diff
@@ -113,7 +113,7 @@ restrict_max_mem={{ sap_hana_deployment_hana_mem_restrict }}
 max_mem={{ sap_hana_deployment_hana_max_mem }}
 
 # Certificate Host Names
-certificates_hostmap=
+certificates_hostmap={{ sap_hana_deployment_certificates_hostmap }}
```

Comment:
I can't see `sap_hana_deployment_certificates_hostmap` defined anywhere or as part of the README

Reply:
Added